composer.json

```diff
@@ -11,6 +11,7 @@
         "php": ">=8.1",
         "doctrine/orm": "^2.0 || ^3.0",
         "doctrine/persistence": "^2.0 || ^3.0",
+        "ramsey/uuid": "^4.7",
```

Comment:
Use https://symfony.com/doc/current/components/uid.html instead. Only because it's probably already installed on a lot of end user applications

Reply:
👍